Server/src/main/java/jettyServer/resources/AccountsResource/AccountsResource.java

```diff
@@ -0,0 +1,69 @@
+package jettyServer.resources.AccountsResource;
+
+import accounts.AccountsClient;
+import accounts.pojos.AccountData;
+import accounts.pojos.CreateAccountRequest;
+import exceptions.DbAccessException;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.InternalServerErrorException;
+import javax.ws.rs.NotAuthorizedException;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+
+@Path("/")
+public class AccountsResource {
```

Comment:
So I guess you want to expose `accounts-service` through the "Server" - was not required, but good idea